app/src/main/java/org/simple/clinic/recentpatient/RecentPatientRelativeTimeStampGenerator.kt

```diff
@@ -0,0 +1,50 @@
+package org.simple.clinic.recentpatient
+
+import android.content.Context
+import org.simple.clinic.R
+import org.threeten.bp.LocalDate
+import org.threeten.bp.ZoneOffset
+import org.threeten.bp.format.DateTimeFormatter
+import java.util.Locale
+import javax.inject.Inject
+
+class RecentPatientRelativeTimeStampGenerator @Inject constructor() {
+
+  fun generate(today: LocalDate, date: LocalDate): RelativeTimestamp {
+    val yesterday = today.minusDays(1)
+
+    return when (date) {
+      today -> Today
+      yesterday -> Yesterday
+      else -> OlderThanTwoDays(date)
+    }
+  }
+
+  fun generate(date: LocalDate): RelativeTimestamp {
+    val today = LocalDate.now(ZoneOffset.UTC)
```

Comment:
This should be generated using the system time zone, not the UTC zone.

Reply:
done